packages/plugins/rubygem-foreman_vault/rubygem-foreman_vault.spec

```diff
@@ -105,5 +105,11 @@ cp -pa .%{gem_dir}/* \
 exit 0
 
 %changelog
+* Mon Mar 16 2020  0.1.0-1
+- Update to 0.1.0
+
+* Tue Jan 07 2020 Eric D. Helms <ericdhelms@gmail.com> - 0.0.1-2
+- Drop migrate, seed and restart posttans
```

Comment:
This part wasn't dropped here, and shouldn't be in 1.24. Mind dropping this log?

Reply:
Yes, will drop it! :-)